.gitignore

```diff
@@ -57,6 +57,8 @@ env/
 
 # Test data
 *.rst.aux.xml
+langegracht/*
```

Comment:
this was added, this to avoid that if a user runs the notebook 'getting started/api.ipynb' it commits the results to the repo. Can you comment?

Reply:
Yes, that is a good idea.